package.json

```diff
@@ -235,6 +235,7 @@
     "cross-fetch": "^2.2.6",
     "trim-newlines": "^3.0.1",
     "aws-sdk": "^2.1141.0",
+    "aws-cdk-lib": "2.129.0",
```

Comment:
>What is the purpose of this declaration, and why is it pinned rather than compatible?

This is to ensure we are resolving to the updated version of the library in our `yarn.lock` even if some of the downstream dependencies like the Gen-1 `amplify-cli-core` are still pulling in the older version. I believe we can remove this override after CLI is upgraded too.